update-log/v3.0.0.md

```diff
@@ -0,0 +1,98 @@
+# Solana Exporter v3.0.0
+
+## Key Changes
+
+The new `solana-exporter` (renamed from `solana_exporter`) contains many new metrics, standardised naming conventions 
+and more configurability.
+
+## What's Changed
+### Metric Updates
+#### New Metrics
+
+Below is a list of newly added metrics (see the [README](https://github.com/asymmetric-research/solana-exporter/blob/master/README.md) 
```

Comment:
```suggestion
Below is a list of newly added metrics (see the [README](../README.md) 
```

Relative paths should work in markdown. You don't need the `../` if you change it to `CHANGELOG.md` in the root however.